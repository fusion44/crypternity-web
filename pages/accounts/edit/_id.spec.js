import { shallow } from "@vue/test-utils"
import index from "./_id"

describe("Account Edit by ID", () => {
  it("renders the correct markup", () => {
    let wrapper = shallow(index)

    let html = wrapper.html()
    expect(html).toMatchSnapshot()
  })
})
