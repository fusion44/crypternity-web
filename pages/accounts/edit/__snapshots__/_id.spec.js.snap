// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Account Edit by ID renders the correct markup 1`] = `
<div>
    <v-layout column="column" justify-center="justify-center" align-center="align-center">
        <v-flex xs12="xs12" sm8="sm8" md6="md6">
            <v-text-field label="Name" hint="e.g. My Binance account"></v-text-field>
            <v-text-field label="API Key"></v-text-field>
            <v-text-field label="API secret"></v-text-field>
            <v-btn>SUBMIT </v-btn>
        </v-flex>
    </v-layout>
</div>
`;
